examples/FirebaseNeoPixel_ESP8266/FirebaseNeoPixel_ESP8266.ino

```diff
@@ -0,0 +1,65 @@
+//
```

Comment:
If the file change too much, git will consider it's a new file (see http://stackoverflow.com/a/433142)